### DISCUSSION TITLE
left-handed feature!

### DISCUSSION THREAD

**Discussion by ivana67dj** [keiwando/toonsquid#60] [Ideas]
hello! is it possible to add a left-handed interface to ToonSquid (i.e. the ability to put all the controls to the right side)

**Comment by LoneStarz** [comment]
Hm I agree! I’m left handed myself this feature would be useful

**Comment by keiwando** [comment]
Hi @ivana67dj! Yes, this will most definitely be one of the first accessibility-related options that I will be adding. Thanks for bringing it up!

Would it be enough to swap the position of the toolbar and the sidebar that is currently on the right or are there more places in the UI (e.g. the timeline or other menus) where swapping the direction of certain parts would benefit left-handed users? (Everybody coming across this is of course welcome to respond, e.g. @LoneStarz)

**Comment by keiwando** [comment]
The 1.10 update now has a left-handed UI mode:
https://toonsquid.com/handbook/settings/appearance/#left-handed

Sorry it took so long and thanks for the patience!
